src/components/menu/MenuItem.tsx

```diff
@@ -24,6 +23,17 @@ const MenuItem: FC<IProps> = ({ item, modifier, isMobile, handleAction }): React
     }
   };
 
+  if (title === 'theme' && typeof modifier[title] === 'string') {
+    const mode = modifier[title].toString();
```

Comment:
а зачем нам приводить к стринге, если выше есть проверка, что это должно быть стринга?

Reply:
согласен, выглядит странно, но тс ругался почему то на то что если он будет булеан то не сработает .toUpperCase()

Reply:
Лучше убрать тему из модификаторов и передавать отдельно. Либо точнее затипизировать `modifier`, то есть указать, что там не просто ключи — строки, а значения — строки или булеаны, а перечислить конкретные свойства. Правда, после того придётся делать проверки на `title` рядом с проверкой на `action`, иначе будет примерно такая ошибка:
```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ sound: boolean; vibration: boolean; theme: string; }'.`
``